web/db/migrate/20150122220754_create_articles.rb

```diff
@@ -1,10 +0,0 @@
-class CreateArticles < ActiveRecord::Migration
```

Comment:
Articles and Bloggs were identical. They were conflicting when I was trying to run the migration on a new set up.